dev/breeze/src/airflow_breeze/commands/kubernetes_commands.py

```diff
@@ -1223,30 +1224,53 @@ def k9s(python: str, kubernetes_version: str, k9s_args: tuple[str, ...]):
     if not k9s_editor:
         env["K9S_EDITOR"] = env["EDITOR"]
     kubeconfig_file = get_kubeconfig_file(python=python, kubernetes_version=kubernetes_version)
-    result = run_command(
-        [
-            "docker",
-            "run",
-            "--rm",
-            "-it",
-            "--network",
-            "host",
-            "-e",
-            "EDITOR",
-            "-e",
-            "K9S_EDITOR",
-            "-v",
-            f"{kubeconfig_file}:/root/.kube/config",
-            "quay.io/derailed/k9s",
-            "--namespace",
-            HELM_AIRFLOW_NAMESPACE,
-            *k9s_args,
-        ],
-        env=env,
-        check=False,
-    )
-    if result.returncode != 0:
-        sys.exit(result.returncode)
+    # Until https://github.com/kubernetes-sigs/kind/pull/3511 is merged and released, running AMD images
+    # on ARM is broken with kind cluster running, so we need to run k9s directly on the host
+    arch, _ = get_host_architecture()
+    if arch != Architecture.ARM:
+        result = run_command(
+            [
+                "docker",
+                "run",
+                "--rm",
+                "-it",
+                "--network",
+                "host",
+                "-e",
+                "EDITOR",
+                "-e",
+                "K9S_EDITOR",
+                "-v",
+                f"{kubeconfig_file}:/root/.kube/config",
+                "quay.io/derailed/k9s",
```

Comment:
```suggestion
                "derailed/k9s",
```

Seems like k9s not updated for a while on `quay.io`:
- https://quay.io/repository/derailed/k9s?tab=tags
- https://github.com/derailed/k9s/issues/2084

In docker hub it has the latest released version
- https://hub.docker.com/r/derailed/k9s/tags

There is still linux/amd image, so I guess it not fix a problem with arm

Reply:
Yeah. But good to use the dockerhub one :)